README.md

```diff
@@ -376,6 +378,50 @@ curl -XPUT https://xxxxxxx.execute-api.us-east-1.amazonaws.com/dev/dynamodb/<has
  -H 'Content-Type:application/json'
 ```
 
+
+### EventBridge
+
+Sample syntax for EventBridge proxy in `serverless.yml`.
+
+```yaml
+custom:
+  apiGatewayServiceProxies:
+    - eventbridge: # source and detailType is set to apigateway requestid by default
+        path: /eventbridge
+        method: post
+        eventBusName: { Ref: 'YourBusName' }
+        cors: true
+    - eventbridge:  # source and detailType are hardcoded
+        path: /eventbridge
+        method: post
+        source: 'hardcoded_source'
+        detailType: 'hardcoded_detailType'
+        eventBusName: { Ref: 'YourBusName' }
+        cors: true
```

Comment:
Can `detail` property be defined here? The property is needed for EventBridge as well.

Reply:
`detail` is required but it is usually the actual json message, so I think it should be passed as body rather than a parameter in spec file. Not sure if it is possible to make body required via plugin config.

Reply:
@jcortega
>we can create a optional parameter detail which points to a property of input.body (request body). And if not given, defaults to input.body. 

Thank you for the suggestion! I prefer this option. Could you please update the code?

Reply:
@horike37 sorry for the delay. I've added detail param as bodyParam.